internal/dashboard/handler/wrapper.go

```diff
@@ -34,8 +35,30 @@ func logHandlerError(c *gin.Context, err error) {
 		Error("handler error")
 }
 
-func Wrap[T any](h Handler[T]) gin.HandlerFunc {
+func Wrap[T any](h Handler[T], funcs ...acbiz.EnforceFunc) gin.HandlerFunc {
```

Comment:
what's the main purpose to support passing multiple enforce functions, is it meant to describe a more strict permission by combining multiple expressions or a more loose policy if any condition fulfilled

Reply:
It's a more strict pattern. By now there is only one enforcer used in practice.